### PR TITLE
fix(contenteditable): fix contenteditable layout width & add onPaste event

### LINKED ISSUE #3153: Labelling - Cursor dismissed when clicking account title during editing label
**Describe the bug**
Clicking anywhere during label editing results in cursor dismissal and needs to be confirmed or 

**Info:**
 - Suite Version: app/web, commit 
 - Browser: , version
 - OS: 
 - Firmware Version: model , revision 

**Steps to reproduce:**
Prerequisite - labelling enabled.
1. Select an arbitrary account and select edit label
2. Click into the account name field (instead of typing),
3. Type the new label title.

**Expected result:**
Cursor is displayed and label editing stays active until it is confirmed, dismissed or navigated to another screen.

**Actual result:**
Cursor is hidden and editing can not be resumed after clicking anywhere during editing.

**Screenshots:**
Screen-Recording-2020-12-08-at-1 (1)

### PR DESCRIPTION
**Pull request by dimasivan**
fix #3153 
also pasting text was not working properly, not sure if there's an issue

### CODE REVIEW

File: packages/suite/src/components/suite/Labeling/components/Metadata/withEditable.tsx [hunk 1]
```diff
@@ -113,6 +113,10 @@ export const withEditable =
             };
 
             window.addEventListener('keydown', keyboardHandler, false);
+            window.addEventListener(
```
Comment (mroz22) [inline packages/suite/src/components/suite/Labeling/components/Metadata/withEditable.tsx]: this listener should be deregistered on component unmount also

**Review by mroz22 — CHANGES_REQUESTED** [review]

**Comment by mroz22** [comment]
also it makes cursor jumping to the end everytime I click somewhere within the label. Feels maybe even more confusing than current behavior :smile_cat:

**Comment by dimasivan** [comment]
> also it makes cursor jumping to the end everytime I click somewhere within the label. Feels maybe even more confusing than current behavior smile_cat

yeah, but you can use arrows for navigating within the label. Currently if you  click anywhere, even input field, it becomes inactive

File: packages/suite/src/components/suite/Labeling/components/Metadata/withEditable.tsx (hunk 1 above)
Comment (dimasivan) [reply packages/suite/src/components/suite/Labeling/components/Metadata/withEditable.tsx]: 54657bd

**Review by mroz22 — APPROVED** [review]
works correctly.

**Pull request merged**